src/eval/cast.rs

```diff
@@ -291,6 +292,14 @@ impl CastInfo {
             msg.push_str(", found ");
             msg.push_str(found.type_name());
         }
+        if_chain::if_chain! {
+            if let Value::Int(i) = found;
+            if parts.contains(&EcoString::from(Value::Length(Length::zero()).type_name()));
```

Comment:
```suggestion
            if parts.iter().any(|p| p == Length::TYPE_NAME);
```
maybe `p.as_str()` if necessary

Reply:
I was looking for `Length::TYPE_NAME` thanks!